.github/workflows/ci.yml

```diff
@@ -74,7 +74,7 @@ jobs:
           -e GITHUB_HEAD_REF \
           -e GITHUB_SHA \
           -e GITHUB_RUN_ID \
-          --memory 7500m \
+          --memory 6500m \
           --memory-swap 7500m \
```

Comment:
```suggestion
          --memory-swap 6500m \
```

Reply:
Why do we not want the runner to use swap memory?

Reply:
There is very little hard disk space, and it's better for the test suite to fail than to swap (because swapping will just make it take longer).

The goal is for all the tests to fit in memory.